<launch>
  <node pkg="diff_drive_calibrated" type="node" name="diff_drive_calibrated">
    <param name="drive_wheel_diameter"        value="0.066"/>
    <param name="drive_wheel_base"            value="0.100"/>
    <param name="drive_wheel_ratio"           value="1.000"/>
    <param name="drive_wheel_encoder_ticks"   value="128"/>
    <param name="stasis_wheel_enabled"        value="false"/>

    <remap from="tick_velocity" to="cmd_ticks"/>
  </node>
</launch>
